org.musesequencer.Muse3.json

```diff
@@ -0,0 +1,201 @@
+{
+  "app-id": "org.musesequencer.Muse3",
+  "branch": "stable",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.11",
```

Comment:
Tried 5.12?

Reply:
As I said on the other PR, jack doesn't build on 5.12

Reply:
https://gitlab.com/freedesktop-sdk/freedesktop-sdk/issues/608

Reply:
As in the other PR, let's embed db and try switching to 5.12.

Reply:
Updated to 5.12 now